### PR TITLE
Fix typo on SDP name

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by fluffy** [comment]
Fixes #130

**Comment by stefhak** [comment]
LGTM!

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
     <section title="Semantics and Syntax" anchor="sec.semantics-and-syntax">
       <section title="Signaling Model" anchor="sec.signaling-model">
         <t>JSEP does not specify a particular signaling model or state
-        machine, other than the generic need to exchange SDP media
+        machine, other than the generic need to exchange SDP session
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: SDP and "session description" seem redundant. I would remove "SDP" and just say "session description" here and below.

Comment (stefhak) [inline draft-ietf-rtcweb-jsep.xml]: That would work fine as far as I'm concerned (I was after changing "media description" to "session description" - does not really matter if there is "SDP" before or not).

Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: I agree with Justin here.

**Pull request merged**